### PR TITLE
Return OpenTofu with version in response

### PR DESCRIPTION
**Pull request by baixinsui**
Related to https://github.com/eclipse-xpanse/xpanse/pull/2034

### CODE REVIEW

File: src/main/java/org/eclipse/xpanse/tofu/maker/models/plan/OpenTofuPlan.java [hunk 1]
```diff
@@ -21,6 +21,6 @@ public class OpenTofuPlan {
     @Schema(description = "OpenTofu plan as a JSON string")
     String plan;
 
-    @Schema(description = "The exact version of the OpenTofu which executed the scripts.")
-    String openTofuVersion;
+    @Schema(description = "The OpenTofu with version which executed the scripts.")
+    String openTofuWithVersion;
```
Comment (swaroopar) [inline src/main/java/org/eclipse/xpanse/tofu/maker/models/plan/OpenTofuPlan.java:25]: OpenTofuVersionUsed
The version of the Opentofu binary used to execute scripts

File: src/main/java/org/eclipse/xpanse/tofu/maker/models/response/OpenTofuResult.java [hunk 2]
```diff
@@ -35,4 +33,6 @@ public class OpenTofuResult {
     @Schema(description = "Data of all other files generated by the openTofu execution."
             + "The map key contains the file name and value is the file contents as string.")
     private Map<String, String> importantFileContentMap;
+    @Schema(description = "OpenTofu with version which executed the command.")
```
Comment (swaroopar) [inline src/main/java/org/eclipse/xpanse/tofu/maker/models/response/OpenTofuResult.java:36]: OpenTofuVersionUsed
The version of the Opentofu binary used to execute scripts

File: src/main/java/org/eclipse/xpanse/tofu/maker/models/validation/OpenTofuValidationResult.java [hunk 3]
```diff
@@ -16,7 +18,13 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class OpenTofuValidationResult {
 
+    @NotNull
+    @Schema(description = "Defines if the scripts is valid.")
     private boolean valid;
-    private String openTofuVersion;
+
+    @Schema(description = "OpenTofu with version which executed the command.")
```
Comment (swaroopar) [inline src/main/java/org/eclipse/xpanse/tofu/maker/models/validation/OpenTofuValidationResult.java:25]: OpenTofuVersionUsed
The version of the Opentofu binary used to execute scripts

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Comment by Alice1319** [comment]
LGTM

**Pull request merged**